imports/ui_3/stylesheets/style.less

```diff
@@ -7,8 +7,8 @@
 @import "ticket.less";
 @import "reports.less";
 
-// Marking the required fields on autoforms
-[data-required] label:after {
+// Marking the required fields on autoforms - only top level in form groups, avoiding marking each checkbox option
+[data-required] > label:after {
```

Comment:
Hogy ez mekkora. Eskuszom nem is emlekeztem mar hogy ezt a csillagozast en csinaltam meg az autoform-hoz. Azt hittem az autoform-ban kell ezt javitani :)